dist/todo_final.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+//this array will contain all the todo list items
+let todoItems = [];
+function listAllTodos() {
+    for (var i = 0; i < todoItems.length; i++) {
+        console.log(todoItems[i].id + " " + todoItems[i].text);
+    }
+}
+/* This function will create a new todo object based on the
+text that was entered in the text input, and push it into
+the `todoItems` array*/
+function addTodo(text, dueAtstr) {
+    let todo = {
+        text,
+        checked: false,
+        id: Date.now(),
+        createdAt: new Date(),
+        dueAt: new Date(dueAtstr), //string to date
+    };
+    todoItems.push(todo);
+    return todo.id;
+}
```

Comment:
Generated JS files shouldn't be checked in to git. Add the `dist/` folder to the `.gitignore`, then follow this tutorial to remove the JS files from git: https://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/